modules/kn-trigger-list.adoc

```diff
@@ -6,12 +6,12 @@
 [id="kn-trigger-list_{context}"]
 = Listing triggers by using the Knative CLI
 
-The `kn trigger list` command prints a list of available triggers.
+Using the `kn` CLI to list triggers provides a streamlined and intuitive user interface. You can use the `kn trigger list` command to list existing triggers in your cluster by using the `kn` CLI.
```

Comment:
These two sentences seem to say the same thing.

Reply:
tweaked a bit to make it less repetitve, thanks